framework/source/class/qx/ui/form/VirtualSelectBox.js

```diff
@@ -70,6 +74,22 @@ qx.Class.define("qx.ui.form.VirtualSelectBox",
       init : 120
     },
 
+    incrementalSearch :
+    {
+      apply : '__applyIncrementalSearch',
+      init : false,
+      check : "Boolean"
+    },
+
+    highlightStyle :
+    {
+        nullable : true,
+        apply : '__applyHighlightStyle',
+        init : {
+            color      : '#FF0000',
+            fontWeight : 'bold'
+        }
+    },
```

Comment:
So this does not seem to get used anymore ?

Reply:
Indeed, thanks. Removed.